jokusoramame/utils.py

```diff
@@ -66,3 +67,14 @@ def get_apikeys(name: str) -> APIKey:
     data.pop("_comment", None)
 
     return APIKey(**data)
+
+
+def chunked(sequence: Sequence[Any], chunk_size: int):
```

Comment:
Typehint return this please, even if it is a sucky typehint.

(I think it's `Generator[Any, None, None]`)